### DISCUSSION TITLE
Milvusdb/milvus: v2.2.11 Frequent service interruptions

### DISCUSSION THREAD

**Discussion by opendatamarket** [milvus-io/milvus#25831] [Q&A and General discussion]
Service interruptions often occur in testing environments running on cloud servers. Can the problem be identified through logs?

<img width="1536" alt="image" src="https://github.com/milvus-io/milvus/assets/106378749/c4cf31f3-b773-43e3-a410-d9f99b9e83df">
<img width="1145" alt="image" src="https://github.com/milvus-io/milvus/assets/106378749/6c61ed26-4500-4c51-abca-5cf250d572e7">

**Comment by xiaofan-luan** [comment]
Did you check if the service has enough memory?
And any error/warn message on why service stopped?
